cluster-with-nodes/modules/downstream-k3s-nodes/files/worker_userdata.tmpl

```diff
@@ -1,5 +1,5 @@
 #cloud-config
-ssh_authorized_keys: 
+ssh_authorized_keys:
   - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDaitXW9CBM/yxEhVaBQ9WEB8KEdwbzlMk6M2URfbr07mA9Rk3ssWf2hS3RMwF76rasRLmrjsge1Q9cc8uck722vs8qCpwz/aMnsHF8kfnifkv3JKgfYCTkVzbsW7HzNmmkbsBVPS2tlVDpG+cHQNwZMgo1MGVJqAvPnrKlLzSj/XVeBKYV8a7/GLwrJTiXegitkyE3i1L42aMwzGDwEDnKkFBYW1JXXim0j97ztWy8YaScuptqd9WJ5NfHgUGfhFSYKlEX+FP5x0oyLVgGRmoUh7NellALQ9mWW+tJgSd7oWPoeJtHLAVMoiX02w1OtBUOFq1s0IJ9nDdU/I9v+grP chriskim@Endeavor
```

Comment:
You might want to take this ssh key out.

Reply:
This was actually here long before we started maintaining this repo 🤷‍♂️. To be honest, this module can probably be removed once we make a module that can create dynamic multi-role/multi-node configurations.